src/plugins/intel_gpu/docs/gpu_plugin_ops_enabling.md

```diff
@@ -54,64 +54,62 @@
-    * Please also review the [general guideline of test infrastructure](https://github.com/openvinotoolkit/openvino/wiki/InferenceEngineTestsInfrastructure)
+    * Also, review the [general guideline of test infrastructure](https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_PLUGIN_DG/PluginTesting.md).
 
-1. [Optional] If there are existing IRs with this operation, try to run the full model(s) to be sure that it's correctly processed within the context
+1. [Optional] If there are existing IRs with this operation, try to run the full model(s) to be sure that it is correctly processed within the context.
 
-1. [Optional] If there are existing IRs with this operation, try to run the full model(s) and estimate performance impact from this operation on total model execution time
+1. [Optional] If there are existing IRs with this operation, try to run the full model(s) and estimate performance impact from this operation on total model execution time.
 
-1. Create PR with your changes
+1. Create a PR with your changes.
     * If you are `OpenVINO` group member in github, CI will be triggered.
-    * Please review the [OpenVINO contribution guide](https://github.com/openvinotoolkit/openvino/wiki/Contribute).
-
-<br>
+    * Review the [OpenVINO contribution guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md).
 
 ## Adding new kernel for an existing primitive
-* The process is quite similar to previous one. You can skip already existing steps.
-* Main work is adding new kernel and registering it from kernel selector.
-* You may need to add unit test for that new kernel. Specific kernel can be chosen with `build_option::force_implementations`.
-* It is not possible to specify kernel from functional test(IE).
 
-<br>
+* The process is quite similar to the previous one. You can skip already existing steps.
+* Main work is adding a new kernel and registering it from the kernel selector.
+* You may need to add a unit test for that new kernel. Specific kernel can be chosen with `build_option::force_implementations`.
+* It is not possible to specify a kernel from a functional test(IE).
 
 ## Writing OCL kernel
 
 ### Jitter
-In GPU OCL kernels, many conditional statements are processed with `#ifdef` so that it can be handled during compile-time. The definitions are created with `jitter.cpp`. It is set during graph compilation. You can see generated macros following the steps in [source dumps](https://github.com/openvinotoolkit/openvino/wiki/GPUPluginDebugUtils#sources-dumps).
+
+In GPU OCL kernels, many conditional statements are processed with `#ifdef` so that it can be handled during compile-time. The definitions are created with `jitter.cpp`. It is set during graph compilation. You can see generated macros, following the steps in [source dumps](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/docs/gpu_debug_utils.md#sources-dumps).
```

Comment:
```suggestion
In GPU OCL kernels, many conditional statements are processed with `#ifdef` so that they can be handled during compile-time. The definitions are created with `jitter.cpp`. It is set during graph compilation. You can see generated macros, following the steps in [source dumps](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/docs/gpu_debug_utils.md#sources-dumps).
```